lib/resque/worker.rb

```diff
@@ -813,17 +813,11 @@ def solaris_worker_pids
     #   RESQUE_PROCLINE_PREFIXresque-VERSION: STRING
     def procline(string)
       $0 = "#{ENV['RESQUE_PROCLINE_PREFIX']}resque-#{Resque::Version}: #{string}"
-      log! $0
+      log :debug, $0
     end
 
-    # Log a message to Resque.logger
-    # can't use alias_method since info/debug are private methods
-    def log(message)
-      info(message)
-    end
-
-    def log!(message)
-      debug(message)
+    def log(severity, message)
```

Comment:
So, I like this change, but this bit of information is swapped out at the moment: was this a public interface? By changing this, do we break things? I'm not sure off the top of my head. I don't have time to dig into this right now, but that's the only question I have about this PR, otherwise looks great :+1: 

(I am too used to Rust and static typing and am now slightly paranoid :sweat:)